lib/utils.js

```diff
@@ -29,18 +29,53 @@ const path = require('path');
 const electron = require('electron');
 const app = electron.app || electron.remote.app;
 
+// define the defaultDataPath and current directory
+const defaultBasePath = 'storage';
+let currentDirectory = '';
+
+/**
+ * @summary Parse directory to avoid parent directories
+ * @function
+ * @private
+ *
+ * @returns {Strings} parsed user data path without parent directories
+ */
+function getDirectory(directory){
+  const directoryArray = directory.split(path.posix.sep);
+  let newDirectory = "";
+  for(var dir of directoryArray){
+    if(dir !== ".."){
+      newDirectory += dir + "/";
+    }
+  }
+  return newDirectory;
+}
+
+/**
+ * @summary Set default user data directory path
+ * @function
+ * @public
+ *
+ * @example
+ * utils.getUserDataPath('newDirectory');
+ */
+exports.setUserDataPath = function(directory) {
+  currentDirectory = getDirectory(directory);
+};
+
 /**
- * @summary Get user data directory path
+ * @summary Get default user data directory path
```

Comment:
Well, its not really the default one, given that the result depends on the directory the user set. Right?